File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-06/plagiarized/L4/09/Level4.ts
import { Scanner } from 'readline-sync';

function proses() {
  const in: Scanner = new Scanner();
  const angka: number[] = new Array(10);

  for (let a = 0; a < 10; a++) {
    console.log('Read a number: ');
    angka[a] = in.nextInt();
  }

  for (let b = 9; b >= 0; b--) {
    console.log(angka[b]);
  }
}

function main() {
  proses();
}

main();

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-06/plagiarized/L4/09/Level4.kt
package IR

import java.util.*

/*
* To change this license header, choose License Headers in Project Properties.
* To change this template file, choose Tools | Templates
* and open the template in the editor.
*/

/**
 *
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
object Level4 {
    @JvmStatic
    fun main(args: Array<String>) {
        proses()
    }

    fun proses() {
        val `in` = Scanner(System.`in`)
        val angka = IntArray(10)
        for (a in 0..9) {
            print("Read a number: ")
            angka[a] = `in`.nextInt()
        }

        for (b in 9 downTo 0) {
            println(angka[b])
        }
    }
}
